validate_env.py

```
@@ -1,74 +1,102 @@
-
-        if resp.status_code == 200:
-            return True, "Anthropic Brain Online"
-        elif resp.status_code == 401:
-            return False, "Anthropic API Key Invalid (401)"
-        return False, f"Anthropic API Error: {resp.status_code}"
+        client = OpenAI(
+            api_key=api_key,
+            base_url="https://api.perplexity.ai"
+        )
+
+        response = client.chat.completions.create(
```

Ruff (0.14.10)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)